agent/roles/manifests/tasks/install-config.yml

```diff
@@ -32,3 +32,15 @@
     src: "agent-config_bond_yaml.j2"
     dest: "{{ install_path }}/agent-config.yaml"
   when: agent_bond_config != 'none'
+
+- name: create add-node directory
```

Comment:
That's not really part of the install-config, it must be placed into its own task file related to add nodes